ocp-templates/app-interface/flower.yaml

```diff
@@ -0,0 +1,69 @@
+# Template for ibutsu prod server
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: ibutsu-flower-template
+# ===============================================
+# Flower
+# ===============================================
+objects:
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    labels:
+      app: ${APP_NAME}
+    name: celery-flower
+  spec:
+    replicas: 1
+    selector:
+      deploymentConfig: celery-flower
+    strategy:
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          app: ${APP_NAME}
+          deploymentConfig: celery-flower
+      spec:
+        containers:
+        - env:
+          # TOOD: use redis secret to set this
+          - name: BROKER_URL
+            value: redis://:${REDIS_PASSWORD}@redis.${NAMESPACE}.svc
+          image: celery-flower
+          imagePullPolicy: IfNotPresent
+          name: quay.io/ibutsu/celery-flower:${IMAGE_TAG}
```

Comment:
```suggestion
          image: quay.io/ibutsu/flower:${IMAGE_TAG}
          imagePullPolicy: Always
```

Reply:
Adjusted image name but the pull policy should stay at 'IfNotPresent' -- it's a recommended best practice for AppSRE